### PR TITLE
DRUP-1147: #ready style and font changes to meet contrast accessibili…

### PR DESCRIPTION
**Pull request by emb03**
…ty guidelines

@z3cka plz review/merge/copy to www-test (this week?) so that WST start review process by Library management. thx!

### CODE REVIEW

File: www/sites/all/themes/uclalib_omega/sass/variables/_colors.scss [hunk 1]
```diff
@@ -1,17 +1,18 @@
 
 // Primary design colors
-$ucla-blue: #3283be;
+$ucla-blue: #315d91;
```
Comment (z3cka) [inline www/sites/all/themes/uclalib_omega/sass/variables/_colors.scss]: I noticed this blue is not on the Color Palette _( http://brand.ucla.edu/wp-content/uploads/ucla-color-palette-v10.pdf )_ from the brand colors, I like your blue color choice better, I was just wondering if we have to stick to the brand colors or not; do we?

File: www/sites/all/themes/uclalib_omega/sass/variables/_colors.scss [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
 $ucla-lt-blue: #84b5d8;
 $ucla-sky-blue: #b5d1e6;
 $ucla-yellow: #fdb930;
+$ucla-accent_color: #fae301;
```
Comment (z3cka) [inline www/sites/all/themes/uclalib_omega/sass/variables/_colors.scss]: ...same with this gold, any reason you chose #fae301 over #ffe800, the suggestion on http://brand.ucla.edu/wp-content/uploads/ucla-color-palette-v10.pdf _(not a critique, just curious)_

**Comment by emb03** [comment]
@z3cka let me change those, I could find the hex codes so I use eyedropper tool and copied off ucla.edu website

**Comment by emb03** [comment]
@z3cka What changes?

> I'm also going to make a few format changes to clean things up, if you don't mind.

File: www/sites/all/themes/uclalib_omega/sass/components/_jquery-ui-tabs.scss [hunk 3]
```diff
@@ -41,26 +42,31 @@
...
-      border-color: $ucla-lt-blue;
+      background: $ucla-gray;
+      border-color: $ucla-gray;
       margin: 0 1px 0 0;
       padding: 0;
       font-weight: 100;
       font-size: 19px;
       a {
-        color: white;
+//        color: white;
```
Comment (z3cka) [inline www/sites/all/themes/uclalib_omega/sass/components/_jquery-ui-tabs.scss]: @emb03 can we remove this? or will SAS freakout with and empty selector sections like
```
a {

}
```
?

**Comment by z3cka** [comment]
will deploy now

**Comment by z3cka** [comment]
`Copying of code from here to www-test2.library.ucla.edu is complete.`

**Pull request merged**